.changeset/cold-drinks-tickle.md

```diff
@@ -0,0 +1,7 @@
+---
+"@aws-amplify/ui": patch
```

Comment:
Do we need to bump ui-react too? And actually ui-angular and ui-view so they can get these CSS changes?

Reply:
If we bump ui, it will make angular/vue/react to have an "Updated dependency" entry in the changelog (like this screenshot from an old version packages pr), so I think we are good with just bumping ui and don't need to explicitly add the others.

<img width="354" alt="Screenshot 2023-07-13 at 10 04 55 AM" src="https://github.com/aws-amplify/amplify-ui/assets/376920/fa53b8f1-399e-4071-8c5a-a9de3c4b1fb4">